<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	font: 900 13pt &quot;Impact&quot;;
}
QMainWindow
{
	background-color: rgb(157, 157, 157)
}

QPushButton
{
  border: 2px solid gray;
  border-radius: 20px;
  padding: 0 4px;
	background-color: rgb(50, 101, 198);
	font: 900 13pt &quot;Impact&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="spriteLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>469</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string>         (Use your mouse to draw in this box and create a sprite)</string>
    </property>
   </widget>
   <widget class="QLabel" name="imagesLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>160</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="gridSizeLayout">
     <item>
      <widget class="QLabel" name="chooseGridSizeLabel">
       <property name="styleSheet">
        <string notr="true">font: 900 13pt &quot;Impact&quot;;</string>
       </property>
       <property name="text">
        <string>Choose Grid Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="twoGridSizeButton">
       <property name="text">
        <string>2x2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fourGridSizeButton">
       <property name="text">
        <string>4x4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eightGridSizeButton">
       <property name="text">
        <string>8x8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sixteenGridSizeButton">
       <property name="text">
        <string>16x16</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="thirtyTwoGridSizeButton">
       <property name="text">
        <string>32x32</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>181</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="newFrameButton">
       <property name="text">
        <string> + Frame</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/frame.png</normaloff>:/Icons/frame.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fpsLabel">
       <property name="styleSheet">
        <string notr="true">font: 900 13pt &quot;Impact&quot;;</string>
       </property>
       <property name="text">
        <string>FPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="fpsSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="previewButton">
       <property name="text">
        <string>Preview Frames</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>160</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="colorLayout">
     <item>
      <widget class="QCheckBox" name="mirrorBox">
       <property name="styleSheet">
        <string notr="true">font: 900 13pt &quot;Impact&quot;;</string>
       </property>
       <property name="text">
        <string>Mirror</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>167</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="gridLabelLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Double-Click to Draw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="gridSizeLabel">
       <property name="styleSheet">
        <string notr="true">font: 900 13pt &quot;Impact&quot;;</string>
       </property>
       <property name="text">
        <string>Grid Size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>171</width>
      <height>280</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="toolsLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Current Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="colorButton">
       <property name="text">
        <string>Color</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/color.png</normaloff>:/Icons/color.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fillButton">
       <property name="text">
        <string>Fill</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/fill.png</normaloff>:/Icons/fill.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invertColorButton">
       <property name="text">
        <string>Invert Colors</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/invert.png</normaloff>:/Icons/invert.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPNGButton">
       <property name="text">
        <string>Export as PNG</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/export.png</normaloff>:/Icons/export.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/load.png</normaloff>:/Icons/load.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/save.png</normaloff>:/Icons/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="spritelyresources.qrc">
         <normaloff>:/Icons/clear.png</normaloff>:/Icons/clear.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpritely">
    <property name="title">
     <string>Spritely</string>
    </property>
   </widget>
   <addaction name="menuSpritely"/>
  </widget>
 </widget>
 <resources>
  <include location="spritelyresources.qrc"/>
 </resources>
 <connections/>
</ui>
